packages/sync/src/modules/Module.php

```diff
@@ -373,4 +384,67 @@ public function get_objects_by_id( $object_type, $ids ) {
 
 		return $objects;
 	}
+
+	/**
+	 * Gets a list of minimum and maximum object ids for each batch based on the given batch size.
+	 *
+	 * @access public
+	 *
+	 * @param int $batch_size The batch size for objects.
+	 *
+	 * @return array An array of min and max ids for each batch.
+	 */
+	public function get_min_max_object_ids_for_batches( $batch_size ) {
+		global $wpdb;
+		$results     = array();
+		$table       = $wpdb->{$this->name()};
```

Comment:
This feels a bit dangerous. What if a class that implements a custom sync module but doesn't correspond to a db table tries to use it? Maybe we should check against a whitelist of tables.

Reply:
Good call, fixed!